README.md

```diff
@@ -1,13 +1,24 @@
 # Express API Project
 
-Replace this readme with your own information about your project. 
+This is a backend project with a purpose to build an API with hardcoded data in a json-file. And create endpoints and using a RESTful approach. The Data used is data over netflix titles both movies and tv shows.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+## Features
 
-## The problem
+- /endpoints = a overview of the endpoints by imported express-list-endpoints
+- /netflix-titles = returning the whole array or data avalible
+- /netflix-titles/movies endpoint that returns only movie titles using filter
+- /netflix-titles/tvshows endpoint that returns only tvshow titles using filter
+- /netflix-titles/year/:year that makes it possible to search and filter the response total data array by a specific year
+- /netflix-titles/show_id/:show_id that makes it possible to find a specific movie or tv show based on a unique show id
+- /netflix-titles/title/:title that makes it possible to search for a title by a specific title through params
+- /netflix-titles?title={title of the movie} endpoint that makes it possible to search title through query
```

Comment:
Documenting the endpoints in the README is really useful. I got inspired by this!